packages/venia-ui/lib/components/MyAccount/ResetPassword/resetPassword.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect } from 'react';
+import { shape, string } from 'prop-types';
+import { Form } from 'informed';
+
+import { useToasts } from '@magento/peregrine';
+import { useResetPassword } from '@magento/peregrine/lib/talons/MyAccount/useResetPassword';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import { Title } from '../../Head';
+
+import Button from '../../Button';
+import FormErrors from '../../FormError';
+
+import resetPasswordOperations from './resetPassword.gql';
+
+import defaultClasses from './resetPassword.css';
+import Password from '../../Password';
+
+const PAGE_TITLE = `Reset Password`;
+
+const ResetPassword = props => {
+    const { classes: propClasses } = props;
+    const classes = mergeClasses(defaultClasses, propClasses);
+    const talonProps = useResetPassword({
+        ...resetPasswordOperations
+    });
+    const {
+        hasCompleted,
+        loading,
+        email,
+        token,
+        formErrors,
+        handleSubmit
+    } = talonProps;
+
+    const tokenMissing = (
+        <div className={classes.invalidTokenContainer}>
+            <div className={classes.invalidToken}>
+                {'Uh oh, something went wrong. Check the link or try again.'}
+            </div>
+        </div>
+    );
+
+    const [, { addToast }] = useToasts();
+
+    useEffect(() => {
+        if (hasCompleted) {
+            addToast({
+                type: 'info',
+                message: 'Your new password has been saved.',
+                timeout: 5000
+            });
+        }
+    }, [addToast, hasCompleted]);
```

Comment:
Did we want both a toast and for the component to render a success message? Feels redundant.

Reply:
Yeah the UX demo had both the message and the toast.